zad5/config/routes.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'artists/index'
```

Comment:
ta linijka jest zbędna ponieważ `resources`, który jest w linii 11 tworzy już 7 ścieżek CRUDowych:
https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions